julearn/model_selection/tests/test_available_searchers.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from sklearn.model_selection._search_successive_halving import (
```

Comment:
Don't see this file. And why is this a private module (`_search_succesive_halving`)

Reply:
I did not find a not used Searcher except for `HalvingGridSearchCV` and I was not able to import it directly from sklearn.model_selection` locally. This is why I did it this way. 
But I exchanged it not with normal `GridSearchCV` and the test work this way as well.
Seems like I over-complicated it before.